lib/matplotlib/dviread.pyi

```diff
@@ -56,12 +56,13 @@ class Dvi:
 class DviFont:
     texname: bytes
     size: float
-    widths: list[int]
```

Comment:
This change belongs in the first commit?

Reply:
woops, fixed.